### ISSUE TITLE
Issue changing views?

### ISSUE THREAD

**Issue by dannysmc95** [SimulatedGREG/electron-vue#235]
### Found an issue or bug? Tell me all about!
It's not an issues it's more of a question as I am quite new to VueJS.

##### Describe the issue / bug.
I am struggling to figure out how to change views? So I created a view in: `app/src/renderer/routes.js` like so:
`{
  	path: '/test',
  	name: 'test',
  	component: require('components/TestView')
  },`

I then created the TestView component the exact same way as the LandingPageView. Swapped all the code to match the correct component it is. Now I have no way of changing page? I can't use something like window.location? Is there a file I need to create to enable this? What am I missing exactly to allow me to have more than one page?

##### How can I reproduce this problem?
Explained above.

##### If visual, provide a screenshot.
![screen shot 2017-06-01 at 17 35 35](https://cloud.githubusercontent.com/assets/5339861/26690343/d51f98de-46f0-11e7-9f30-b336434d660d.png)

##### Tell me about your development environment.
  * Node version: V7.10.0
  * NPM version: 4.2.0
  * vue-cli version: 2.8.2
  * Operating System: Mac OS X El Capitan

**Comment by dannysmc95** [comment]
I apologise in advanced if I am doing something wrong or I am missing something, I just need a few different views, so I can do different things in each view?

**Comment by SimulatedGREG** [comment]
Make sure to read up on the `vue-router` documentation for any questions. electron-vue docs only cover a general overview of the original project structure. 

https://router.vuejs.org/en/api/router-link.html
https://router.vuejs.org/en/essentials/navigation.html

**Comment by dannysmc95** [comment]
Awesome thanks for the help.

**Comment by dannysmc95** [comment]
Can I ask one more question, I was looking at how to make it go back to the root view?

So your example, I wanted to go back to the landing-page view. It doesn't seem to let me though. I tried using
`<router-view to="/">Home</router-view>`
but that doesn't work, I tried creating a redirect in the router called home and doing `to="home'` but every time I do it, it doesn't render? Any ideas? Looked at the documents but can't find anything.
